WWTWebservices/ConfigurationManagerFilePlateTilePyramid.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Configuration;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace WWTWebservices
+{
+    public class ConfigurationManagerFilePlateTilePyramid : IPlateTilePyramid
+    {
+        private readonly RegexDictionary _plateNamePath;
+
+        public ConfigurationManagerFilePlateTilePyramid()
+        {
+            _plateNamePath = new RegexDictionary
+            {
+                { @"dssterrapixel\.plate", "WWTTilesDir" },
+                { @"DSSpngL5to12_x(\d+)_y(\d+)\.plate", "DssTerapixelDir" },
```

Comment:
I'm imagining that it might get a little tedious to update this dictionary for every potential plate file in the core data collection. Maybe it would be simpler for the `GetStream` API to take a `localDirectoryPrefix` argument that the Azure implementation can just ignore?

Reply:
Sure. I've added that prefix and it'll be ignored on the azure implementation.